pkg/validation/limits.go

```diff
@@ -223,6 +225,11 @@ func (l *Limits) RegisterFlags(f *flag.FlagSet) {
 	_ = l.MinShardingLookback.Set("0s")
 	f.Var(&l.MinShardingLookback, "frontend.min-sharding-lookback", "Limit queries that can be sharded. Queries within the time range of now and now minus this sharding lookback are not sharded. The default value of 0s disables the lookback, causing sharding of all queries at all times.")
 
+	_ = l.MaxQueryBytesRead.Set("0B")
+	f.Var(&l.MaxQueryBytesRead, "frontend.max-query-bytes-read", "TODO: Max number of bytes a query would fetch")
+	_ = l.MaxSubqueryBytesRead.Set("0B")
```

Comment:
```suggestion
```
nit: `0B` is the default zero value when unset

Reply:
Thank you!